Makefile

```diff
@@ -0,0 +1,9 @@
+build:
+	cargo build
+
+run:
+	cargo run
```

Comment:
Do we need to lock the rust version? Add a toolchain file?

Reply:
We can add it later.